### DISCUSSION TITLE
Don't open

### DISCUSSION THREAD

**Discussion by Sahinoglu0** [mashb1t/Fooocus#77] [Q&A] [answered]
![image](https://github.com/user-attachments/assets/81eddc4a-c01c-48a8-810d-6b293f54b6f6)
It opens, and after a short while it gives an error and closes. What should I do?

**Comment by mashb1t** [comment]
I don't speak turkish (?), can you please translate the error message?
You also only have 1024MB of VRAM. Ensure you meet the minimum system requirements, see readme.

**Reply by Sahinoglu0** [reply]
The application failed to initialize properly. [0x**]
My system is 
R5 5600G 
RX 6600 
16 GB RAM

**Reply by mashb1t** [reply]
Did you follow the AMD setup instructions and have you completed all steps?

**Reply by Sahinoglu0** [reply]
![image](https://github.com/user-attachments/assets/20192993-7cd3-4c00-a11d-d82d86251e16)
Yes, I implemented that.

**Reply by mashb1t** [reply, marked as answer]
as i don't have an AMD GPU i can't support here. Please check the open issues on lllyasviel/fooocus (upstream) and keep in mind that i've stepped down as official maintainer of Fooocus, see https://github.com/lllyasviel/Fooocus/discussions/3504. The community can probably help you anyways.

**Reply by Sahinoglu0** [reply]
I have another question. We changed the run.bat code for AMD GPU. When I run it, it deletes and re-downloads something.
Do I need to re-download them every time to start?
Can't it be started in a shorter way?

**Reply by mashb1t** [reply]
you only have to run this code once, i'd propose to copy your run.bat to install_and_run.bat and revert your run.bat to the version before

**Reply by Sahinoglu0** [reply]
Where is the location of the install_and_run.bat file? Do I create it myself?

**Reply by mashb1t** [reply]
> copy your run.bat to install_and_run.bat and revert your run.bat to the version before

as mentioned
